src/components/PopupWithConfirm.js

```diff
@@ -0,0 +1,40 @@
+import Popup from "./Popup.js";
+
+export default class PopupWithConfirm extends Popup {
+  constructor(popupSelector) {
+    super({ popupSelector });
+    this._popupForm = this._popupElement.querySelector(".modal__form");
+    this._submitButton = this._popupForm.querySelector(
+      ".form__button_type_delete"
+    );
+    // this._submitButtonText = this._submitButton.textContent;
```

Comment:
* See here http://joxi.ru/zAN3J9VtwgdBVr . After the 1st confirmation the button text disappears


Please, uncomment this line. It's needed for `setLoading`